custom_components/hacs/update.py

```diff
@@ -148,9 +149,15 @@ async def async_release_notes(self) -> str | None:
                 self.repository.data.last_version = next(iter(self.repository.data.published_tags))
 
         release_notes = ""
-        if len(self.repository.releases.objects) > 0:
-            release = self.repository.releases.objects[0]
-            release_notes += release.body
+        if any([r for r in self.repository.releases.objects if r.tag_name == self.installed_version]):
+            new_releases = dropwhile(lambda r: r.tag_name != self.installed_version, reversed(self.repository.releases.objects))
+            for release in new_releases:
+                release_notes += f"# {release.name}\n\n"
```

Comment:
```suggestion
                release_notes += f"# {release.tag_name}\n\n"
```
`.name` can be added as context if it does not equal the `.tag_name`